### PR TITLE
Migrate ESRP signing tasks to v5

### LINKED ISSUE #101: Migrate ESRPCodeSigning to v5
https://github.com/microsoft/kiota-serialization-cbor-dotnet/blob/c65b7062518aea50672a7001734cc80bbd52d2bf/.azure-pipelines/ci-build.yml#L153

Example https://github.com/microsoft/kiota-dotnet/pull/320

### PR DESCRIPTION
**Pull request by timayabi2020**
Fixes #101

### CODE REVIEW

File: .azure-pipelines/ci-build.yml [hunk 1]
```diff
@@ -17,288 +17,300 @@ pool:
   vmImage: windows-latest
 
 variables:
-  buildPlatform: 'Any CPU'
-  buildConfiguration: 'Release'
+  buildPlatform: "Any CPU"
```
Comment (andrueastman) [inline .azure-pipelines/ci-build.yml:20]: Any chance the change in indentation across the file is intended? Are you able to confirm this does not break the pipeline?

**Comment by andrueastman** [comment]
Could we also take the opportunity to run the migration tool to have this pipeline on a governed template as well?

File: .azure-pipelines/ci-build.yml (hunk 1 above)
Comment (timayabi2020) [reply .azure-pipelines/ci-build.yml:20]: I am not sure what has tampered with the indentation but looks like all checks passed

**Comment by timayabi2020** [comment]
> Could we also take the opportunity to run the migration tool to have this pipeline on a governed template as well?

Here is the issue to track this. #104

**Pull request closed without merging**